dev/bazel/flags.bzl

```diff
@@ -23,6 +23,7 @@ lnx_cc_common_flags = [
     "-Wformat",
     "-Wformat-security",
     "-Wreturn-type",
+    "-Wno-unused-variables",
```

Comment:
The same in makefile, see `dev/make`

Reply:
I think, we don't need this. Unused vars can be handled case by case using `[[maybe_unused]]`
https://en.cppreference.com/w/cpp/language/attributes/maybe_unused